docs/getting-started/live-networks.md

```diff
@@ -86,4 +86,20 @@ You can check the configuration by running the following command:
 regen config
 ```
 
-For more information about using the CLI, see [Command-Line Interface](../regen-ledger/interfaces.md#command-line-interface).
\ No newline at end of file
+For more information about using the CLI, see [Command-Line Interface](../regen-ledger/interfaces.md#command-line-interface).
+
+## Testnet Tokens
+
+In order to interact with the test networks, you'll need some tokens. You can redeem tokens for each test network by executing the following `curl` commands.
+
+*For Redwood Testnet:*
+
+```bash
+curl http://209.182.218.23:8000/faucet/<account_address>
+```
+
+*For Hambach Testnet:*
+
+```bash
+curl http://147.182.238.116:8000/faucet/<account_address>
```

Comment:
@kaustubhkapatral I realize this is not working at the moment but is the plan to reuse the same address once we get hambach back online?

Reply:
Faucet was moved from that to this server `http://185.181.11.150:8000/faucet`. I can reuse this server when we restart hambach.

Reply:
```suggestion
curl http://185.181.11.150:8000/faucet/<account_address>
```

Reply:
@ryanchristo I've assigned domains to sentries of hambach and redwood which also hosts the faucet.

http://redwood-sentry.vitwit.com:8000/faucet/<account_address>

http://hambach-sentry.vitwit.com:8000/faucet/<account_address>